bookops_marc/bib.py

```diff
@@ -563,9 +580,9 @@ def pymarc_record_to_local_bib(record: Record, library: str) -> Bib:
     Returns:
         `bookops_marc.bib.Bib` instance
     """
-    if isinstance(record, Record):
-        bib = Bib(library=library)
-        bib.leader = record.leader
-        bib.fields = record.fields[:]
+    # if isinstance(record, Record):
```

Comment:
I think it is safer to check if given object is `pymarc.Record`. If not, this will raise an exception that somehow will need to be handled down the road, instead of returning `None`. 
Any particular reason for commenting it out?

Reply:
I think I was trying to get rid of mypy's missing return statement error but looking at it again there are ways to fix it without raising an exception.
We can revert this change or could change it to:
`if isinstance(record, Record):
    ...
    return bib
else:
    return None`
There will be a typing error with either option.

Reply:
Explicit is better than implicit (I haven't read The Zen of Python, but heard the phrase often enough;-)) - please change to if ... else to explicitly return None, and push to the PR. A function without a return statement returns by default None.

Reply:
I will take a look and try to fix typing problem.

Reply:
The solution turned to be trivial: the return type was changed from `-> str` to `-> Optional[str]`